tutorials/themes/tutorials-theme/theme.conf

```diff
@@ -44,7 +44,7 @@ globaltoc_includehidden = true
 
 # HTML navbar class (Default: "navbar") to attach to <div> element.
 # For black navbar, do "navbar navbar-inverse"
-navbar_class = navbar
+navbar_class = navbar navbar-inverse
```

Comment:
Can you explain this addition?

Reply:
We have a dark navbar, so navbar-inverse supposedly should be added. Just in case.